### ISSUE TITLE
POLYGON omits isolated purely-horizontal or purely-vertical line segments

### ISSUE THREAD

**Issue by AndrewWittenberg** [NOAA-PMEL/Ferret#1918]
Since Ferret v7.13 dated 7/6/2017, the POLYGON command no longer renders isolated purely-horizontal or purely-vertical line segments:

```
 	NOAA/PMEL TMAP
 	FERRET v7.44 (optimized)
 	Linux 2.6.32-696.30.1.el6.x86_64 64-bit - 12/10/18
 	20-Mar-19 15:40     

yes? let xpts = {2,8,,2,8,,2,2.1,,8,8}
yes? let ypts = {2,2.1,,8,8,,2,8,,2,8}
yes? poly/hl=0:10/vl=0:10/line=14 xpts, ypts
yes? plot/ov/vs/line=16 xpts+1, ypts+1
```
![v744](https://user-images.githubusercontent.com/34082818/54714168-b809c800-4b26-11e9-80bf-f9f3771ffad6.gif)

The bug is in both the latest Ferret and PyFerret.  It was not in v7.12 dated 5/16/2017, which produced this:

![v712](https://user-images.githubusercontent.com/34082818/54714158-b6400480-4b26-11e9-9af0-35468c5b2c69.gif)

This bug is messing with our longstanding utility scripts that use POLYGON to produce lines and/or shapes.

**Comment by AnsleyManke** [comment]
This behavior changes between Ferret v7.1 and v7.2

**Comment by AndrewWittenberg** [comment]
Between 7.12 and 7.13.

**Comment by AnsleyManke** [comment]
This comes from the fix for PyFerret bug #1814, where a filled polygon with zero extent causes the graphics call to hang.  It turns out that that check needs to be applied only for FILLED polygons, so it's simple enough to change the test and let the lines be drawn, skipping over the degenerate filled polygons.
